source/_patterns/_shame.scss

```diff
@@ -178,14 +178,16 @@ _:-ms-fullscreen,
   }
 }
 
-.teaser--size-m .teaser__prekicker--fixed-for-ie strong,
-.teaser--size-l .teaser__prekicker--fixed-for-ie strong,
-.teaser--size-xl .teaser__prekicker--fixed-for-ie strong {
+.teaser--size-m .teaser__prekicker--fixed-for-ie,
+.teaser--size-l .teaser__prekicker--fixed-for-ie,
+.teaser--size-xl .teaser__prekicker--fixed-for-ie {
```

Comment:
Nimm doch .teaser__prekicker--fixed-for-ie auch noch mit rein ;)

(und warum bei size-s nicht?